modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding_clusteroperator_conditiontypes_{context}"]
+= Understanding cluster operator condition types
+
+The status of cluster operators is revealed with the condition type, informing you of the current state of your operator's health. The following definitions cover a list of some common ClusterOperator condition types. Some operators may have additional condition types with operator-specific language, which are not included here. 
+
+Cluster version operator is responsible for collecting these status conditions from cluster operators and represents these is an way which will be easier for OCP administrators to understand the state of the OCP cluster.
```

Comment:
Operator stuff plus ISG recommends against using statements like "user-friendly" or "easy to understand", so the below edit also includes a suggestion to avoid this:
```suggestion
The Cluster Version Operator (CVO) is responsible for collecting the status conditions from cluster Operators so cluster administrators can better understand the state of the {product-title} cluster.
```